### ISSUE TITLE
[Bug]: vocalisation of number of items selected #172

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#266]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

Samsung S20

### OS Version

Android 11

### App version

1.0

### Describe the issue

the update of the number of tabs is not vocalised (following the click on plus or minus)

### Expected Behavior

when user change in the number of tabs (following the click on plus or minus), and Talkback is activated, this update should be vocalised correctly.
